python/ipywidgets/ipywidgets/embed.py

```diff
@@ -12,8 +12,8 @@
 
 import json
 import re
-import ipywidgets.widgets.widget
 from .widgets import Widget, DOMWidget
+from .widgets.widget import _instances
```

Comment:
I don't like importing variables because modifying them in the original will not modify the copy here (something I do in Solara).

Reply:
I pushed a new version that imports the widget module instead. Better?